### DISCUSSION TITLE
Setting max_plots to 0 or not setting at all will prevent manager to start

### DISCUSSION THREAD

**Discussion by MaciejGorczyca** [swar/Swar-Chia-Plot-Manager#351] [Q&A]
Hey, changing max_plots to 0 or not setting that value (to make it go forever until disks fill) will cause error in commands.py at line 53. The error says to look for manager.log but that one is empty. Changing max_plots to anything higher than 0 will make the manager work fine. Using dev version.

**Comment by swar** [comment]
I mean you set max_plots to 0. That means you don't want any plots...

Please use Github Discussions next time. This is not an Issue with the software.

**Comment by MaciejGorczyca** [comment]
> I mean you set max_plots to 0. That means you don't want any plots...

Not setting max_plots to any value doesn't work as well, which means I don't want to LIMIT the number of plots and make it to keep going forever. A bit counter-intuitive. Max plots set to 0 or not set at all should mean it will loop forever imo.

**Reply by swar** [reply]
I don't agree with this because there are valid reasons for wanting to keep your max plots at 0 so no more jobs can get kicked off. I think for now you should keep the number really high and use that as infinite. 9999 will take the best computer almost 5 years to go through.

**Reply by MaciejGorczyca** [reply]
Then setting max_plots should be optional and no value set should be infinite or setting it to -1 should make it infinite. I assume basically infinite loop would be additional code and you can make the similar effect by setting it 9999 as you said so maybe you are right and its not worth the code changes and the effort.
